### PR TITLE
#58_Add submission template

### LINKED ISSUE #58: SDS-009: The CSC OMOP`ProjectManager`
## Description
This issue defines ProjectManager behaviour in the CSC OMOP databse. The purpose of the ProjectManager is to process and validate management tasks as outlined below.

## Linked Issues
#14 
#15 
#30 

## Requirements
1. Must instantiate `ProjectManager` class
2. Must allow `ProjectManager` to perform the tasks outlined in the acceptance criteria

## Acceptance Criteria
- [ ] `ProjectManager` validates new project requests e.g., based on required standardised directory structure, file names, required project-specific YAML file containing mandatory and optional details, etc.
- [ ] The `ProjectManager` can output a one-pager summary of the project's key details e.g., a PDF.
- [ ] #78
- [ ] #77
- [ ] The `ProjectManager` can generate queries e.g., SQL, XNAT API, ElasticSearch i.e., CogStack, etc., that can be run on the service's database component.
- [ ] The 'ProjectManager` has PostgreSQL constraints enabled to enforce naming and schema conventions, including entity relationships.
- [ ] Documentation of details on the functionality and usage of `ProjectManager` is available on the service's core GitHub repository e.g., Read The Docs, GitHub Wiki, etc.

## Testing Guidance
- A pytest must be included for each ProjectManager task
- A pytest for each task must pass

### PR DESCRIPTION
**Pull request by AnilMistry**
## Pull Request Checklist
- [x] Have you read "[How to write the perfect pull request](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)"?
- [x] Have you added the issue reference number as the _first_ thing in your pull request title e.g., `BUG-1234 Fix bug`?
- [x] Have you explicitly selected a reviewer to review your pull request?
- [x] Have you assigned your pull request to yourself and any other people working on this piece of work?

## Linked Issue
partially resolves #58 

## Purpose

- adds data files for OMOP CDM tags to exports, as a .yml for tags which can be turned off. 
- updates the submission.yml for data submitted by user to be parsed and used by the program and for approvals

## Additional Information

The outcomes of a  discussion in the morning of Day 2, confirmed that we wont request the OMOP tags from user. the tags we outpout will be standard unless uses argue they want extra.

### CODE REVIEW

File: templates/template_project_request/submission.yml [hunk 1]
```diff
@@ -1,55 +1,66 @@
...
-opt_out_required: False # If the project type is 'Research', this should automatically be interpreted as 'True'
-qips_approval_number: 12345
-anonymisation required: ['hospitalid', 'dicom tags'] # This should be limited to certain set of options
-team: Radiotherapy # Used for XNAT project aliases e.g., for project requestor grouping, cost codes, etc.
+project_type: Quality Improvement                       # Either 'Research' or 'Quality Improvement'
 
+# Approvals and consent ---------------
+opt_out_required: null                                  # If the project type is 'Research', this should automatically be interpreted as 'True'
+qips_approval_number: 12345
+anonymisation required: ['hospitalid', 'dicom tags']    # This should be limited to certain set of options
```
Comment (mikewoodward94) [inline templates/template_project_request/submission.yml:11]: *anonymisation_required (add in underscore for consistency)

File: templates/template_project_request/submission.yml [hunk 2]
```diff
@@ -1,55 +1,66 @@
...
-- ethnicity_source_value: False
-- ethnicity_source_concept_id: False
+    size: 50
+# - key: ['hospitalid', 'studydate']                     # List of keys on which to base search e.g., hospital IDs, NHS numbers, accession IDs, hospital IDs and study dates, etc.
+    accession_numbers_file: 'cohort/xnat_ingest.csv'
+    inclusion criteria:
+        phrases:
+            - aortic stenosis
+            - aortic valve calcification
+        patient_ids: "cohort/patient_ids.csv",
```
Comment (mikewoodward94) [inline templates/template_project_request/submission.yml:43]: remove the comma

File: templates/template_project_request/submission.yml [hunk 3]
```diff
@@ -1,55 +1,66 @@
...
+                - 456
+        opcs4_codes :
+            - and_or: AND
+              codes:
+                - 123
+                - 456
+        orders:
+            - 'cardiac angiogram - CT'
+            - ''
+        start_date: '2022-12-31',
```
Comment (mikewoodward94) [inline templates/template_project_request/submission.yml:61]: remove the comma

File: templates/template_project_request/submission.yml [hunk 4]
```diff
@@ -1,55 +1,66 @@
...
+        end_date: 'today'}        # Dict of inclusion criteria
```
Comment (mikewoodward94) [inline templates/template_project_request/submission.yml:62]: remove the bracket

**Review by mikewoodward94 — COMMENTED** [review]
Hi Anil,

Just noticed these small things when using this submission template!

**Comment by AnilMistry** [comment]
Corrections made, cheers @mikewoodward94

**Review by heyhaleema — APPROVED** [review]

**Pull request merged**